src/components/SearchModal/CurrencyList/__snapshots__/index.test.tsx.snap

```diff
@@ -171,7 +171,7 @@ exports[`renders currency rows correctly when currencies list is non-empty 1`] =
             </div>
           </div>
           <div
-            class="c10 css-1j6a53a"
+            class="c10 css-yfjwjl"
```

Comment:
This is a job for another PR, but I bet we could configure styled-components to make these names more deterministic. See this section of the docs:

- https://styled-components.com/docs/tooling#better-debugging